lv_conf.h

```diff
@@ -55,7 +56,7 @@
 /* Dot Per Inch: used to initialize default sizes.
  * E.g. a button with width = LV_DPI / 2 -> half inch wide
  * (Not so important, you can adjust it to modify default sizes and spaces)*/
-#define LV_DPI              100     /*[px]*/
+#define LV_DPI              60     /*[px]*/
```

Comment:
This affects all platforms, as far as I know. Can we leave it at 100, or at least condition the change for STM32?

Reply:
Yes we can leave 100.  On my screen the style was better with 60

Reply:
I just checked the examples with 100 and are quite terrible :)